### PR TITLE
Creating the User interface

### PR DESCRIPTION
**Pull request by danmainah**
This milestone entails creating a user_-interface, from where the users will be interacting with the game.

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: INVALID 🔴 
@danmainah Your project is marked invalid due to the fact that i can run it on my machine and when next you want to submit for review make sure you check for linters

**Comment by danmainah** [comment]
> # STATUS: INVALID 🔴
> @danmainah Your project is marked invalid due to the fact that i can run it on my machine and when next you want to submit for review make sure you check for linte

File: bin/main [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  if [1, 2, 3].include? count
+    puts 'Game over!'
+    puts "The winner is #{one} !"
+  elsif [4, 5, 6].include? count
+    puts "The winner is #{two} !"
+  else
+    puts "It's tie! \n Game over"
+  end
+  i += 1
+end
```
Comment (Berabjesus) [inline bin/main]: - [ ] use a terminating variable such as the one below

```
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
**Hi @danmainah this is a really nice start to the project.**
### you do have a few changes to carry out before we proceed so please take a look at the things you have to work on and their respective review comments below.

- [x] Your user interface is repeating the same prompt for player 1 twice when asking for names.
image

- [ ] Use a terminating variable instead of `while i <= 9` in the while loop. Checkout the comment below.

- [x] [OPTIONAL] you still need to have the `lib` folder with a `.gitkeep` file.

## **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

File: bin/main (hunk 1 above)
Comment (danmainah) [reply bin/main]: Thank you for the feedback

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

@danmainah, weldone on this project,changes is required:
- [x] game informs player if the selected move is a draw move (without computing draw). [Example screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/game_tie.png)
- [x] kindly add linters folder in the root of your project, can't still see file

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Changes required:wrench::hammer::full_moon:
**Hi Daniel!
Great job so far:clap::clap:. And thank you for making some of the changes requested by the last reviewer.
However, we still need to make one change to get the project approved:muscle:.**

**To improve**

- [ ] As highlighted by the last reviewer. Add .rubocop.yml config file to the root of the project. It's currently missing. Use this [link](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml).

Make the above-required change and when you are ready, request another review.

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @danmainah ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
HAPPY CODING :computer:
**[Umair Arshad (Code Reviewer)](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**